### PR TITLE
fix for warning in svm test file

### PR DESCRIPTION
**Pull request by pwisniewskimobica**
fix for issue : https://github.com/KhronosGroup/OpenCL-CLHPP/issues/236

### CODE REVIEW

File: tests/test_openclhpp.cpp [hunk 1]
```diff
@@ -4214,10 +4214,10 @@ void clSVMFree_stubForMemoryAllocation(cl_context context, void *svm_pointer,
     int cmock_num_calls)
 {
     TEST_ASSERT_EQUAL_PTR(svm_pointer, testMemory);
-    delete svm_pointer; 
+    delete (int*) svm_pointer; 
 }
 void testSVMMemoryAllocation()
-{
+{//
```
Comment (alycm) [inline tests/test_openclhpp.cpp:4220]:
```suggestion
{
```

Something committed accidentally?

File: tests/test_openclhpp.cpp [hunk 2]
```diff
@@ -4214,10 +4214,10 @@ void clSVMFree_stubForMemoryAllocation(cl_context context, void *svm_pointer,
     int cmock_num_calls)
 {
     TEST_ASSERT_EQUAL_PTR(svm_pointer, testMemory);
-    delete svm_pointer; 
+    delete (int*) svm_pointer;
```
Comment (alycm) [inline tests/test_openclhpp.cpp:4217]:
```suggestion
    delete[] (int*) svm_pointer; 
```

As `testMemory` was allocated with `new[]` should use `delete[]`.  I realise that problem existed before this PR, but touching the line made me notice it.

**Review by alycm — APPROVED** [review]
In general LGTM.

**Review by MathiasMagnus — APPROVED** [review]
Ship it.

**Pull request merged**